### PR TITLE
Pr merging protocol

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
Make additions to README.md

### CODE REVIEW

File: internal/cli/util/utils.go [hunk 1]
```diff
@@ -51,6 +51,11 @@ func RunCommand(t *testing.T, commandString string, maxAttempts int, backoff tim
 	var count int
 	for {
 		count++
+
+		if count == maxAttempts {
+			commandString = strings.Replace(commandString, "--silent", "", 1)
```
Comment (stewartie4) [inline internal/cli/util/utils.go:56]: good idea but this will mess with output assertions right?

Comment (Kishan-Dhakan) [reply internal/cli/util/utils.go]: Yep. I changed it a bit to get us the logging without messing output assertions

Comment (stewartie4) [reply internal/cli/util/utils.go]: slightly concerned that this will lead to very unwieldy log output in the console but we'll see LGTM for now

**Review by stewartie4 — APPROVED** [review]

**Comment by stewartie4** [comment]
Thanks for the detailed README Kishan

**Pull request merged**